esx_service/cli/vmdkops_admin.py

```diff
@@ -46,6 +46,8 @@
 def main():
     log_config.configure()
     kv.init()
+    if not vmdk_ops.is_service_available():
+       sys.exit('Unable to connect to the host-agent on this host, ensure the ESX host agent is running before retrying.')
```

Comment:
This is the admin CLI code, if Hostd is down then admin CLI errors out as below,

/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Unable to connect to the host-agent on this host, ensure the ESX host agent is running before retrying.